modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayManage.macro

```diff
@@ -38,18 +40,32 @@ definition {
 		Click(locator1 = "TestrayManage#CLOSE_MODAL");
 	}
 
+	macro assertCategorieNotPresent {
```

Comment:
Hi @joelrocha68 would you please correct this word? "categorie" must be changed to "category"

assertCategoryNotPresent